tests/unit/test_skill_loader.py

```
@@ -60,18 +58,34 @@ def test_discover_skills_agents_paths_override_legacy_fallbacks_and_preserve_req
 
 
-def test_skill_registry_load_skill_emits_install_steps_for_requirements() -> None:
+def test_skill_registry_load_skill_includes_install_normalization_guidance() -> None:
     registry = SkillRegistry(
         [
             SkillEntry(
                 name="deck",
                 description="Deck skill",
-                body="Make slides.",
+                body="## Dependencies\n\nuv pip install python-pptx",
                 source="/tmp/SKILL.md",
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/SKILL.md"

(S108)